frontend/src/functions/waffle.ts

```diff
@@ -4,7 +4,7 @@ export type RuntimeDataWithWaffle = RuntimeData & {
   WAFFLE_FLAGS: RuntimeData["WAFFLE_FLAGS"];
 };
 
-export function flagIsActive(
+export function isFlagActive(
```

Comment:
I renamed this to be aligned with other function names, but since we don't have an explicit guideline for that anyway, I'm happy to drop this commit.

Reply:
Personally, this makes sense to me. +1 from me.